src/server.js

```diff
@@ -697,6 +697,78 @@ async function startPolkadotAPI() {
         }
     });
 
+    app.get('/api/query/staking/validatorCount', async function (req, res) {
```

Comment:
Kindly reorder these as follows:
1. activeEra
2. eraElectionStatus
3. erasRewardPoints
4. erasStakers
5. erasTotalStake
6. erasValidatorReward
7. forceEra
8. validatorCount

To keep order in line with that of [the docs](https://polkadot.js.org/api/substrate/storage.html#staking)

The same order should also be kept in `substrate_query_interface.js`